cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureUtils.java

```diff
@@ -291,6 +292,25 @@ public List<CloudVmInstanceStatus> deallocateInstances(AuthenticatedContext ac,
         return statusesAfterDeallocate;
     }
 
+    @Retryable(backoff = @Backoff(delay = 1000, multiplier = 2, maxDelay = 10000), maxAttempts = 5)
```

Comment:
Set to a lower value. See the AwsConnector as an example. The intent is to fail fast. In fact, I'm not even sure we want retries here or in AWS ... that can be tracked via a separate PR.

Reply:
Done